Realm/RLMProperty.mm

```diff
@@ -219,12 +223,22 @@ - (BOOL)setTypeFromRawType:(NSString *)rawType {
     static const char setPrefix[] = "@\"RLMSet<";
     static const int setPrefixLen = sizeof(setPrefix) - 1;
 
+    static const char dictionaryPrefix[] = "@\"RLMDictionary<";
+    static const int dictionaryPrefixLen = sizeof(dictionaryPrefix) - 1;
+
     static const char numberPrefix[] = "@\"NSNumber<";
     static const int numberPrefixLen = sizeof(numberPrefix) - 1;
 
     static const char linkingObjectsPrefix[] = "@\"RLMLinkingObjects";
     static const int linkingObjectsPrefixLen = sizeof(linkingObjectsPrefix) - 1;
 
+    auto isCollection = [&]() {
```

Comment:
This is a misleading name for a function which mutates ivars. It also appears to be more code than just not having it?

Reply:
true